templates/Gemfile.erb

```diff
@@ -12,6 +12,7 @@ gem "oj", "3.5.1"
 gem "fluent-plugin-multi-format-parser", "~> 1.0.0"
 gem "fluent-plugin-concat", "~> 2.3.0"
 gem "fluent-plugin-grok-parser", "~> 2.5.0"
+gem "fluent-plugin-prometheus", "1.3.0"
```

Comment:
`~>` is better than fixed number because teeny version is increamented for bug fixes.

Reply:
OK done!